MobiFlight/Config/BaseDevice.cs

```diff
@@ -9,9 +9,10 @@ namespace MobiFlight.Config
     public class BaseDevice : IConfigItem
     {
         protected DeviceType _type = DeviceType.NotSet;
+        protected bool _muxClient = false;
```

Comment:
Why do we need this in the base device class?
Let's move this to the InputMultiplexer class

Reply:
I should go back and check, but I seem to recall that this reference was somehow necessary in order to be able to access the Mux driver data from the device tree entries (the driver is not a device, therefore is not in the tree) AND/OR to identify what devices should carry the driver subpanel, also to find the first one of them, in which the subpanel is made editable.